presto-native-execution/presto_cpp/main/QueryContextManager.cpp

```diff
@@ -62,6 +62,11 @@ std::shared_ptr<core::QueryCtx> QueryContextManager::findOrCreateQueryCtx(
   if (auto queryCtx = lockedCache->get(queryId)) {
     return queryCtx;
   }
+  static constexpr uint64_t kMaxLocalExchangeBufferSizeDefault = 32UL << 20;
```

Comment:
What is this logic for? Shouldn't this be read from the config files? Please, update PR description to explain all the changes in this PR.